### PR TITLE
[Issue # ] Incorporating the Flow to Run Locally with fixes to issues 54, 49, 52

### PR DESCRIPTION
**Pull request by stravid87**
Development has already been merged into stravid87/run_local so there should be no conflicts and it should be a straight forward merge.

To test, see the README.md which lays out how to run the project locally to test it. 

Also included here are Hermann's updates to the UI

### CODE REVIEW

File: server/handlers/tunnel.go [hunk 1]
```diff
@@ -156,8 +154,8 @@ func Tunnel(w http.ResponseWriter, r *http.Request) {
 	fmt.Println(r.URL)    // (http://localhost:5000/api/v1 ) > /api/v1
 	fmt.Println("Ravi Adds Path: ", r.URL.Path)
 
-	// backendURL := fmt.Sprintf("http://localhost:8000%s", r.URL)
-	backendURL := fmt.Sprintf(os.Getenv("VITE_BACKEND")+"%s", r.URL)
+	backendURL := fmt.Sprintf("http://localhost:8000%s", r.URL)
```
Comment (DeeStarks) [inline server/handlers/tunnel.go:157]: I think we should dynamically specify this

**Review by DeeStarks — APPROVED** [review]
LGTM. Just a comment on hard coding the backend URL

File: server/handlers/tunnel.go (hunk 1 above)
Comment (stravid87) [reply server/handlers/tunnel.go:157]: Great point. 
Thankfully a super easy fix.
In fact, makes it simpler overall.

Comment (stravid87) [reply server/handlers/tunnel.go]: Complete with a quick change to `server/handlers/tunnel.go` and `server/.env.dev `

**Pull request merged**